deployment/ansible/roles/deploy_web_app/templates/docker-compose.yml.j2

```diff
@@ -10,25 +10,17 @@ services:
     environment:
       - DB_HOST=db
       - DB_PORT=3306
-      - DB_NAME= mydatabase
-      - DB_USER= {{ db_user }}
-      - DB_PASSWORD= mysecretpassword
-    networks:
-      - app-network
+      - DB_NAME= dbs
```

Comment:
templates  for templating 
DB_NAME={{ DB_NAME }}

etc. etc.

Reply:
partly done